torsiondrive/qm_engine.py

```diff
@@ -327,6 +335,166 @@ def load_native_output(self, filename='output.dat'):
         m.build_topology()
         return m
 
+class EngineGaussian09(QMEngine):
+    def load_input(self, input_file):
+        """
+        !!!only Cartesian molecule specification is supported at the moment!!!
+        Load Gaussian09 input
+        Example input file:
+
+        %Mem=6GB
+        %NProcShared=2
+        %Chk=lig
+        # B3LYP/6-31G(d) Opt=ModRedundant
+
+        water energy
+
+        0   1
+        O  -0.464   0.177   0.0
+        H  -0.464   1.137   0.0
+        H   0.441  -0.143   0.0
+
+
```

Comment:
remove empty lines at the end of doc string?

Reply:
Yes, Gaussian does need some blank lines at the bottom note added.